modules/apps/object/object-web/src/main/resources/META-INF/resources/info/item/renderer/object_entry.jsp

```diff
@@ -34,9 +34,23 @@ for (Map.Entry<String, Serializable> entry : objectEntryValues.entrySet()) {
 				</a>
 			</c:when>
 			<c:otherwise>
-				<span class="text-truncate-inline">
-					<span class="text-truncate"><%= entry.getValue() %></span>
-				</span>
+				<c:choose>
+					<c:when test="<%= entry.getValue() instanceof Link %>">
```

Comment:
Hey @carolmariaabb , I think we also need to cover the scenario where the Object has the Display Page Template (friendlyURL) and there is an attachment type column, this way the other columns will still have the link to redirect to DPT and only the attachment column will have the link to download the file.

Reply:
@guilhermedcamacho I updated the logic. I'm not sure if I understood correctly, but I made the changes considering what you said:

> other columns will still have the link to redirect to DPT and only the attachment column will have the link to download the file